package.json

```diff
@@ -8,8 +8,10 @@
     "android": "expo start --android",
     "ios": "expo start --ios",
     "web": "expo start --web",
-    "lint": "expo lint"
-  },
+    "lint": "expo lint" ,
+    "start:staging": "set APP_ENV=staging && npx expo start",
+    "start:prod": "set APP_ENV=production && npx expo start"
```

Comment:
- let's have a single start command and pick the `APP_ENV` from the environment.